python/cudf/io/orc.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+from cudf.dataframe.dataframe import DataFrame
+
+import pyarrow.orc as orc
+import warnings
+
+
+def read_orc(path, columns=None, **kwargs):
```

Comment:
You specify `**kwargs` but you don't do anything with them.

Reply:
Yea, this is for future compatibility with the PyArrow ORC reader. Happy to remove it for now, but shouldn't break anything.